resources-for-our-team.html

```diff
@@ -18,6 +18,21 @@ <h4>Office Hours &amp; Tech Support</h4>
     <p>English Now!'s office is still open online and by phone - at regularly scheduled hours, Monday through Thursday, 9:00am until 5:00pm; Fridays 9:00am until 1:00pm; and Saturdays 9:30am until 12:00pm.</p>
     <p>Please note: 1) **Please call or text Ioanna on her mobile phone on Monday and Thursday between 2:00 and 4:00pm ONLY**; 2) if you are having a technical issue outside that time, please <u>text the school phone</u> at <a href="tel:+13017183575">+1.301.718.3575</a>. State succinctly what the issue is, and how urgent it is. We may change the protocol for tech support and will inform you if this changes. Thank you so much for your support!</p>
     <br>
+    <h4>Online Teaching: Zoom Protocols and Lessons Learned</h4>
+    <ol>
+        <li><strong>Zoom Basics: Setting Up Zoom and Joining a Class </strong>(For Instructors or Students) Download <a target="_blank" href="files/team-resources/English Now! - Setting Up and Joining a Class on Zoom.pdf">here</a>.</li>
+        <li><strong>Lessons Learned in Online Teaching-Week One. </strong>Download <a target="_blank" href="files/team-resources/English Now! - Lessons Learned in Online Teaching-Week One - 2020.03.20.pdf">here</a>. (thanks, Jennifer, for compiling this!)</li>
+    </ol>
+    <h4>Online Curriculum Resources (watch this space, this list will grow)</h4>
+    <ol>
+        <li>Bookshare, <em>an ebook library that makes reading easier.</em> Members can access a huge collection of ebooks and read their way with the most customizable ebooks for people with reading barriers. URL:&nbsp; <a target="_blank" rel="noopener" href="https://www.bookshare.org/home">https://www.bookshare.org/home</a> Login: <a href="mailto:info@english-now.com">info@english-now.com</a> PW: Englishnow1 (Thanks, Jeannie, for this recommendation!)</li>
+        <li><a target="_blank" rel="noopener" href="https://breakingnewsenglish.com/">Breaking News English</a>: Current events stories - includes mini-lessons with answers and audio available in five speeds.
+        <li><a target="_blank" rel="noopener" href="http://englishpage.com">Englishpage.com:</a> Grammar review and exercises on many topics, great for students to reference independently or use in class
+        <li><a target="_blank" rel="noopener" href="http://readworks.org">Readworks</a>: a free resource for K-12 educators with readings searchable by grade/Lexile level/topic; used in MCPS
+        <li><a target="_blank" rel="noopener" href="https://learningenglish.voanews.com">Voice of America</a>: many short news stories, kept current, with the ability to change the pace of playback sped
+        <li><a target="_blank" rel="noopener" href="http://npr.org">National Public Radio</a>: all your favorite programs, and many stories have transcripts that students can review as prework or together
```

Comment:
Why do you use `rel="noopener"` for all these links and elsewhere? I'm pretty sure you only need `target="_blank"` to open in a new tab.

Reply:
Yeah, google was just telling me to include it. It's just some odd security measure against opening external websites in a new tab.